store/deprecated_flags.go

```diff
@@ -0,0 +1,67 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+)
+
+// All of these flags are deprecated and will be removed in release v2.0.0
+// we leave them here with actions that crash the program to ensure they are not used,
+// and to make it easier for users to find the new flags (instead of silently crashing late during execution
+// because some flag's env var was changed but the user forgot to update it)
+var (
+	DeprecatedFallbackTargetsFlagName = withDeprecatedFlagPrefix("fallback-targets")
+	DeprecatedCacheTargetsFlagName    = withDeprecatedFlagPrefix("cache-targets")
+	DeprecatedConcurrentWriteThreads  = withDeprecatedFlagPrefix("concurrent-write-routines")
+)
+
+func withDeprecatedFlagPrefix(s string) string {
+	return "routing." + s
+}
+
+func withDeprecatedEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_" + s}
+}
+
+// CLIFlags ... used for EigenDA client configuration
+func DeprecatedCLIFlags(envPrefix, category string) []cli.Flag {
+	return []cli.Flag{
+		&cli.StringSliceFlag{
+			Name:     DeprecatedFallbackTargetsFlagName,
+			Usage:    "List of read fallback targets to rollover to if cert can't be read from EigenDA.",
+			Value:    cli.NewStringSlice(),
+			EnvVars:  withDeprecatedEnvPrefix(envPrefix, "FALLBACK_TARGETS"),
+			Category: category,
+			Action: func(*cli.Context, []string) error {
+				return fmt.Errorf("flag --%s (env var %s) is deprecated, use --%s (env var %s) instead",
```

Comment:
(env var %s) will print an array here, consider changing

Reply:
this is done for compliance with the flag declaration - i.e. `StringSliceFlag` --> `[]string` for Action. IIUC its not doing anything with the function params:
```
t=2024-10-29T02:41:26-0400 lvl=crit msg="Application failed" message="flag --routing.cache-targets (env var [EIGENDA_PROXY_CACHE_TARGETS]) is deprecated, use --storage.cache-targets (env var [EIGENDA_PROXY_STORAGE_CACHE_TARGETS]) instead"
```

Reply:
hmmm that's a good point. I see that you changed them back. Agree with your argument here though! Perhaps we should have used "env varS" with an s then instead... but at this point we're bikeshedding either is fine.